client/signup/steps/domains/index.jsx

```diff
@@ -260,6 +260,9 @@ class DomainsStep extends Component {
 			submitSignupStep: this.props.submitSignupStep,
 		} );
 
+		const siteAccentColor =
+			this.props.flowName === 'newsletter' && this.props.queryObject?.siteAccentColor;
```

Comment:
Do we need gating to the newsletter flow necessarily, and just use the prop if its set and ignore otherwise?

Reply:
The suggestion to do so came from @yansern in [this comment](https://github.com/Automattic/wp-calypso/issues/66765#issuecomment-1223811540). I believe it should be fine to not bind it to the 'newsletter' flow only.